core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -993,6 +999,18 @@ class KafkaApis(val requestChannel: RequestChannel,
     }
   }
 
+  def handleApiVersionRequest(request: RequestChannel.Request) {
+    val responseHeader = new ResponseHeader(request.header.correlationId)
+    def isApiVersionRequestVersionSupported: Boolean = {
```

Comment:
I think you want "val" and not "def" here, since this is not a method.
and make sure the style matches our conventions